src/app/pages/Login/Login.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import Button from '../../components/Button/Button';
+import DesginElementLogin from '../../components/DesignElement/DesginElementLogin';
+import Logo from '../../components/Icons/Logo';
+import LabeledInput from '../../components/LabeledInput/LabeledInput';
+import styles from './Login.module.css';
+
+function Login(): JSX.Element {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    alert(`You entered ${email}:${password}`);
+  }
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.overlay}></div>
+      <header className={styles.logo}>
+        <Logo />
+        <span className={styles.logo__text}>
+          it<span className={styles.logo__number}>2</span>gether
+        </span>
+      </header>
+
+      <main className={styles.main}>
+        <form onSubmit={handleSubmit} className={styles.form}>
+          <div>
+            <LabeledInput
+              placeholder="&nbsp;"
```

Comment:
Why is the placeholder a whitespace?

Reply:
The label is used as "placeholder" and "label". But I need the placeholder as a whitespace to do the animation.

Reply:
Ok got it. It looks a bit weird. Maybe add it as default argument in `LabeledInput` to make it possible to add no placeholder.
https://javascript.info/function-basics#default-values

Reply:
Sounds good. Will do the fixes 👍  Thanks